src/main/java/zeenea/connector/common/ItemReference.java

```diff
@@ -10,8 +10,8 @@ public final class ItemReference {
   /** The identifier for the item. */
   @NotNull private final ItemIdentifier itemIdentifier;
 
-  /** The connection reference associated with the item, if any. */
-  @Nullable private final ConnectionReference connectionReference;
+  /** The data source identifier associated with the item */
+  @Nullable private final DataSourceIdentifier dataSource;
```

Comment:
Pour être en accord avec le nommage général, je te propose
```suggestion
  @Nullable private final DataSourceIdentifier dataSourceIdentifier;
```

Reply:
C'est corrigé !